### ISSUE TITLE
Improve Execution Summary

### ISSUE THREAD

**Issue by jordan-evans** [awslabs/aws-service-catalog-puppet#88]
At the end of a Puppet run we get an output that looks something like below

```
===== Luigi Execution Summary ===== 
 
Scheduled 121 tasks of which: 
* 119 ran successfully: 
    - 119 ProvisionProductTask(...) 
* 2 failed: 
    - 2 ProvisionProductTask(...) 
 
This progress looks :( because there were failed tasks 
 
===== Luigi Execution Summary ===== 
```

It would be good if there were any failures, it also briefly summarised which launch & account the failed task occurred in.

**Comment by jordan-evans** [comment]
Just an FYI that this seems to already be done for certain failures (E.g.)
```
2019-06-30T11:44:53.380Z ===== Luigi Execution Summary =====
2019-06-30T11:44:53.380Z 
2019-06-30T11:44:53.380Z Scheduled 117 tasks of which:
2019-06-30T11:44:53.380Z * 115 ran successfully:
2019-06-30T11:44:53.380Z     - 1 CreateAssociationsForPortfolioTask(...)
2019-06-30T11:44:53.380Z     - 1 CreateSpokeLocalPortfolioTask(account_id=012345678910, region=eu-west-2, portfolio=test-base)
2019-06-30T11:44:53.380Z     - 113 ProvisionProductTask(...)
2019-06-30T11:44:53.380Z * 1 failed:
2019-06-30T11:44:53.380Z     - 1 ImportIntoSpokeLocalPortfolioTask(account_id=012345678910, region=eu-west-2, portfolio=test-base, hub_portfolio_id=port-gwr7wrqzepuny)
2019-06-30T11:44:53.380Z * 1 were left pending, among these:
2019-06-30T11:44:53.380Z     * 1 had failed dependencies:
2019-06-30T11:44:53.380Z         - 1 CreateLaunchRoleConstraintsForPortfolio(...)
2019-06-30T11:44:53.380Z 
2019-06-30T11:44:53.380Z This progress looks :( because there were failed tasks
2019-06-30T11:44:53.380Z 
2019-06-30T11:44:53.380Z ===== Luigi Execution Summary =====
```
So is Luigi just cutting the output because it's too large maybe? Any way to configure this?

**Comment by eamonnfaherty** [comment]
When luigi fails, the stderr and some stdout for the task is printed to the console.
When luigi cannot schedule a task it will tell you but not always why.

I am being asked about logging more so I will have a look at getting some log events pushed out to s3.

**Comment by eamonnfaherty** [comment]
this was released in 0.5.0
